internal/librarian/state.go

```diff
@@ -28,10 +29,12 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
-const pipelineStateFile = "state.yaml"
-const pipelineConfigFile = "pipeline-config.json"
-const serviceConfigType = "type"
-const serviceConfigValue = "google.api.Service"
+const (
+	pipelineStateFile  = "state.yaml"
```

Comment:
We have both pipelineStateFile and config.PipelineStateFile now. I think we just need this one and do not need to export it?

Reply:
Done.